src/query/exceptions.hpp

```diff
@@ -211,6 +211,18 @@ class IsolationLevelModificationInMulticommandTxException : public QueryExceptio
       : QueryException("Isolation level cannot be modified in multicommand transactions.") {}
 };
 
+class StorageModeModificationInMulticommandTxException : public QueryException {
+ public:
+  StorageModeModificationInMulticommandTxException()
+      : QueryException("Storage mode cannot be modified in multicommand transactions.") {}
+};
+
+class StorageModeModificationInMultiTxException : public QueryException {
+ public:
+  StorageModeModificationInMultiTxException()
+      : QueryException("Storage mode cannot be modified when multiple transactions are running.") {}
```

Comment:
I would enrich this error message by saying what the user could do next so he/she doesn't end on Discord disturbing Kate :smile: E.g Terminate your transaction by first running show transactions and then ... or a variant of this, I'm sure you will have great ideas

Reply:
Nice!